### DISCUSSION TITLE
Airtable Import Issue [Mar 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5320] [Discord Threads]
> 🙋‍♂️ **yves**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Acedia**  
#### Hi <@1072181536054591580> ,  
#  
> 🙋‍♂️ **Acedia**  
#### I will try to guide you through this issue, is it possible for you to send me a screenshot of import modal or does it stuck?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1072550488412786740)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
